.github/workflows/mvn-verify-check.yml

```diff
@@ -246,8 +251,47 @@ jobs:
           }
           done
 
+  verify-213-modules:
+    needs: set-scala213-versions
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{ fromJSON(needs.set-scala213-versions.outputs.sparkJDK17Versions) }}
+    steps:
+      - uses: actions/checkout@v4 # refs/pull/:prNumber/merge
+
+      - name: Setup Java and Maven Env
+        uses: actions/setup-java@v4
+        with:
+          distribution: adopt
+          java-version: 17
+
+      - name: check runtime before tests
+        run: |
+          env | grep JAVA
+          java -version && mvn --version && echo "ENV JAVA_HOME: $JAVA_HOME, PATH: $PATH"
+
+      - name: Build JDK
+        run: |
+          # https://github.com/NVIDIA/spark-rapids/issues/8847
+          # specify expected versions
+          export JAVA_HOME=${JAVA_HOME_${{ matrix.java-version }}_X64}
+          export PATH=${JAVA_HOME}/bin:${PATH}
+          java -version && mvn --version && echo "ENV JAVA_HOME: $JAVA_HOME, PATH: $PATH"
+          # test command, will retry for 3 times if failed.
+          max_retry=3; delay=30; i=1
+          while true; do
+            mvn verify \
```

Comment:
https://github.com/NVIDIA/spark-rapids/actions/runs/9811144386/job/27146029550
https://github.com/NVIDIA/spark-rapids/actions/runs/9811144386/job/27146028574

this is still building against scala 2.12. please switch to scala2.13 folder before the mvn command https://github.com/NVIDIA/spark-rapids/blob/branch-24.08/.github/workflows/mvn-verify-check.yml#L232-L233